mixins/demo/localize-mixin.html

```diff
@@ -55,14 +59,15 @@ <h2>localize()</h2>
 			</d2l-demo-snippet>
 
 			<h2>localizeHTML()</h2>
-
 			<d2l-demo-snippet>
 				<script type="module">
+import { html, LitElement } from 'lit';
+import { LocalizeDynamicMixin } from '../../mixins/localize-dynamic-mixin.js';
 class Mission extends LocalizeDynamicMixin(LitElement) {
 
 	static get localizeConfig() {
 		const langResources = {
-			'en': { mission: `Transforming the way [a]the world[/a] learns` },
+			'en': { mission: `[em]Transforming[/em] the way [a]the world[/a] learns` },
```

Comment:
```suggestion
			'en': { mission: `[i]Transforming[/i] the way [a]the world[/a] learns` },
```

Reply:
I was just testing it out -- forgot to remove, thanks.